uktrade/data-hub-frontend
# Add an event handler on the adviser filter chips

## Description of change

This enables users to remove filters by clicking on the filter chips.

## Test instructions

1. Go to the investments project page.
2. Type "Barry" or "Harold" in the adviser typeahead, select them and you should see the chips appear in the collection list header. You should then be able to deselect these filters by clicking on the "x" in the chip.

## Screenshots
![Screenshot 2020-11-16 at 09 07 57](https://user-images.githubusercontent.com/10154302/99233897-bd338480-27eb-11eb-8c3b-c5f3dcc26daa.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
